### PR TITLE
feat: initial implementation

### PR DESCRIPTION
**Pull request by SweetOps**
## what
* initial implementation of module

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -1,64 +1,58 @@
...
-
-# List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
+- name: terraform-null-label
+  description: Terraform Module to define a consistent naming convention by (namespace,
+    stage, name, [attributes])
+  url: https://github.com/cloudposse/terraform-null-label
+- name: terraform-cloudflare-zone
+  description:
+  url: https://github.com/cloudposse/terraform-cloudflare-zone
+- name: terraform-cloudflare-waf-rulesets
```
Comment (aknysh) [inline README.yaml]: remove the self-reference

File: README.yaml [hunk 2]
```diff
@@ -67,32 +61,68 @@ description: |-
...
+    name       = "waf"
+    attributes = ["cf"]
+    delimiter  = "-"
+  }
+
+  module "waf_rulesets" {
+    source = "cloudposse/waf-rulesets/cloudflare"
+    # Cloud Posse recommends pinning every module to a specific version
+    # version = "x.x.x"
+    zone = "cloudposse.co"
```
Comment (aknysh) [inline README.yaml:84]:
```suggestion
    
    zone = "testing.cloudposse.co"
```

**Review by aknysh — COMMENTED** [review]
LGTM, please see comments

File: README.yaml [hunk 3]
```diff
@@ -1,64 +1,58 @@
...
-  - name: "Discourse Forum"
-    image: "https://img.shields.io/discourse/https/ask.sweetops.com/posts.svg"
-    url: "https://ask.sweetops.com/"
+- name: Latest Release
+  image: https://img.shields.io/github/release/cloudposse/terraform-cloudflare-waf-rulesets.svg
+  url: https://github.com/cloudposse/terraform-cloudflare-waf-rulesets/releases/latest
+- name: Slack Community
+  image: https://slack.cloudposse.com/badge.svg
+  url: https://slack.cloudposse.com
+- name: Discourse Forum
```
Comment (aknysh) [inline README.yaml]: remove `Discourse Forum` entry

File: README.yaml [hunk 4]
```diff
@@ -67,32 +61,68 @@ description: |-
...
+
+    context = module.label.context
   }
   ```
 
 # Example usage
 examples: |-
   Here is an example of using this module:
-  - [`examples/complete`](https://github.com/cloudposse/terraform-example-module/) - complete example of using this module
+  - [`examples/complete`](https://github.com/cloudposse/terraform-cloudflare-waf-rulesets/) - complete example of using this module
```
Comment (aknysh) [inline README.yaml]:
```suggestion
  - [`examples/complete`](examples/complete) - complete example of using this module
```

File: README.yaml [hunk 5]
```diff
@@ -1,64 +1,52 @@
...
-# Short description of this project
+- name: terraform-provider-cloudflare
+  description: Cloudflare Terraform Provider
+  url: https://registry.terraform.io/providers/cloudflare/cloudflare/latest
 description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
-  use when creating new repositories.
+  Terraform module to manage CloudFlare WAF rulesetes. 
+  
+  NOTE: This module has started as hardfork of https://github.com/innovationnorway/terraform-cloudflare-waf-rulesets with adoption to `ClouPosse` code style.
```
Comment (osterman) [inline README.yaml]:
```suggestion
  __NOTE:__ This module is a hard fork of [Innovation Norway's](https://github.com/innovationnorway/terraform-cloudflare-waf-rulesets) terraform module and adapted to Cloud Posse conventions.
```

**Review by mcalhoun — APPROVED** [review]
LGTM!

**Pull request merged**